src/index.html

```diff
@@ -5,8 +5,257 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,700"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/">
```

Comment:
```suggestion
      <a href="#">
```
that is what I get when click on the logo:
![image](https://github.com/mate-academy/layout_catalog/assets/86923089/7f95b30a-be7d-45ee-b5f4-3de93b437d0e)